README.md

```diff
@@ -77,9 +77,9 @@ Password: admin
 
 Support the following ways currently.
 
-- [Source Codes](./docs/en/latest/deploy.md)
+- [Source Codes](./docs/en/latest/install.md#source-source)
 - [Docker](./docs/en/latest/deploy-with-docker.md)
-- [RPM package（only for CentOS 7）](./docs/en/latest/deploy-with-rpm.md)
+- [RPM package（only for CentOS 7）](./docs/en/latest/install.md#rpm-rpm)
```

Comment:
```suggestion
- [RPM, Docker, Source Codes](./docs/en/latest/install.md)
- [Rebuild docker image](./docs/en/latest/deploy-with-docker.md)
```

Reply:
Agree, great😄Would it be better to swap the position of Docker and RPM and keep the order of the file titles？

Reply:
Sure, that's my mistake, I will change the suggested content.

Reply:
I show here: `Applying suggestions on deleted lines is currently not supported.`

Reply:
@juzhiyuan @bzp2010 emmm, I can't Commit suggestion now😐Can you operate it?

Reply:
![image](https://user-images.githubusercontent.com/2106987/142125386-a1f9e112-f812-4b40-bb67-82b156179a60.png)

The seem warning 🤔 Could you update this PR manually?

Reply:
😮I have updated once, you can see it?